standalone/inc/pup/PUPManager.cpp

```diff
@@ -104,11 +112,35 @@ void PUPManager::LoadConfig(const string& szRomName)
    return;
 }
 
-const string& PUPManager::GetRootPath()
+void PUPManager::LoadPlaylists()
 {
-   if (!m_init) {
-      PLOGW.printf("Getting root path before initialization");
```

Comment:
In pup packs the root path is sometimes accessed before the player is initialized. This used to return nullpointers to the vbscript, now it's initialized in the constructor and this warning is no longer required.